docs/user/merging-metadata.md

```diff
@@ -0,0 +1,93 @@
+# Merging metadata {#merging-metadata}
+
+## About merging metadata {#about-merging-metadata}
+
+## How to Merge Indicators {#indicators-merge}
+
+The indicator merge feature lets you combine multiple source indicators into a single target indicator. Follow these steps to merge indicators:
+
+1. **Go to the Indicators List:**
+   Click on **Indicators -> Indicator** in the left-hand side (LHS) menu.
```

Comment:
Is there a reason why we add the `LHS` abbreviation? It doesn't seem like the abbreviation is used again later, so it seems unnecessary?